cms/service/ProxyService.py

```diff
@@ -488,6 +489,11 @@ def submission_scored(self, submission_id):
                             submission_id)
                 return
 
+            # ScoringService sent us a submission of another contest, they
+            # do not know about our contest_id in multicontest setup.
+            if submission.task.contest_id != self.contest_id:
+                return
```

Comment:
Would you like to put a `info`/`debug` message here? This could help with troubleshooting. Same for `submission_tokened()` below.